### PR TITLE
Page error

### PR DESCRIPTION
**Pull request by AxelHumbert**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1206575137133723/1206623525445233) by [Unito](https://www.unito.io)

### CODE REVIEW

File: src/front/src/pages/Error.tsx [hunk 1]
```diff
@@ -1,7 +1,35 @@
+import * as React from 'react';
+import {Grid} from "@mui/material";
+import Box from "@mui/material/Box";
 
 function Error() {
 
-    return (<h1>Error</h1>);
+    return (
+        <Box>
+            <Grid alignContent={"center"}>
```
Comment (tvelu77) [inline src/front/src/pages/Error.tsx:9]: Is this CSS directly in a tag ? I think you should use CSS classes ?

**Review by tvelu77 — APPROVED** [review]
It is good, just check if you can use classes instead of giving directly css into tag

File: src/front/src/pages/Error.tsx (hunk 1 above)
Comment (AxelHumbert) [reply src/front/src/pages/Error.tsx:9]: This isn't possible with this configuration of React server, because we don't take styles folder for the build

**Review by Adrien-COSTANDI — APPROVED** [review]
good for me

**Pull request merged**